server/routes/dashboard/index.ts

```diff
@@ -293,9 +282,16 @@ export default function dashboard(): Router {
       })
     }
 
+    const showLocationFilter = userCaseloadIds.length > 1 || permissions.hasPecsAccess
     let tableHead: HeaderCell[] | undefined
     let paginationParams: LegacyPagination
     if (reportsResponse) {
+      let tableColumns: ColumnEntry[]
```

Comment:
Could this be a good case for the ternary operator:

```JavaScript
      const tableColumns: ColumnEntry[] = showLocationFilter ? multiCaseloadColumns : singleCaseloadColumns
```

Reply:
yeah that’s a good idea. i just moved this chunk without changes

Reply:
7df97b4186fd4512a8b00bb4828b251d9617b3bf